test/dbTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require '../db.php';
+use PHPUnit\Framework\TestCase;
+class dbTest extends TestCase {
+    public function testSuccessLoop(){
+        $array = [['name' => 'simon', 'designer' => 'versace', 'style' => 'bomber', 'year_released' => 2000, 'image' => 'image.jpg']];
+        $expected = '<div><img src="image.jpg" alt="garment image"></div><div>simon</div><div>versace</div><div>bomber</div><div>2000</div>';
+        $inputA = $array;
+        $case = arraySplit($inputA);
+        $this->assertEquals($expected, $case);
+    }
+    public function testFailureLoop(){
```

Comment:
If the database contains no entries (i.e. getItems() returns an empty array) the page errors.
Is it worth building in a failure test for this?Is there a way of making sure that it fails in a controlled way, rather than a fatal error?

Reply:
Very true, I should add more conditions to db.php to make sure that the correct information is parsed instead of an error.

Reply:
Great- looks like what you've added to the function should work (if an empty array is input) but could do with creating a unit test to check this